src/fu-main.c

```diff
@@ -1345,6 +1340,10 @@ main (int argc, char *argv[])
 	else if (timed_exit)
 		g_timeout_add_seconds (5, fu_main_timed_exit_cb, priv->loop);
 
+	lang = g_getenv ("LANG");
+	if (lang != NULL)
+		g_debug ("Started with locale %s", lang);
```

Comment:
I'd just say `g_debug ("Started with locale %s", g_getenv ("LANG"));` -- then if LANG is unset we set that too.